release/sample-service.yaml

```diff
@@ -0,0 +1,17 @@
+name: 'sample-service'
+version: 'v1.2.0'
+releaseName: 'geneva'
+releaseStream: 'master'
+repo: 'https://github.com/edgexfoundry/sample-service.git'
+gitTag: true
+gitTagDestination: 'https://github.com/edgexfoundry/sample-service.git'
+dockerImages: true
+dockerSource:
+  - 'https://nexus3.edgexfoundry.org/..'
+  - 'https://nexus3.edgexfoundry.org/..'
```

Comment:
why do we have two instances of this dockerSource?

Reply:
It is meant to be a list of docker images to release for the repository. All of the repositories in EdgeX have more than one docker image to release.